src/styles/theme.js

```diff
@@ -1,11 +1,17 @@
 const theme = {
     colors: {
         white: "white",
-        opacWhite: "rgba(255, 255, 255, 0.5)",
-        lightGrey: "#e8e8e8",
         grey: "#e0e1e2",
+        lightGrey: "#e8e8e8",
+        opacWhite: "rgba(255, 255, 255, 0.5)",
         primary: "#051d2d",
-        secondary: "#ffad00"
+        secondary: "#ffad00",
+        messageBoxBackground: "#f8ffff",
```

Comment:
We better follow naming convention here: describe the colour instead of the usage. _(ie`neutralBackground`)_